en/core-utility-libraries/email.rst

```diff
@@ -166,8 +171,6 @@ The following configuration keys are used:
 - ``'attachments'``: List of files to attach. See ``CakeEmail::attachments()``.
 - ``'emailFormat'``: Format of email (html, text or both). See ``CakeEmail::emailFormat()``.
 - ``'transport'``: Transport name. See ``CakeEmail::transport()``.
-- ``'log'``: Log level to log the email headers and message. ``true`` will use
-  LOG_DEBUG. See also ``CakeLog::write()``
```

Comment:
You're right that this is the one funny config option. I've never found the need to change it at runtime which is why there is no method.